rules/misc_rules.build_defs

```diff
@@ -230,6 +230,28 @@ def gentest(name:str, test_cmd:str|list|dict, labels:list&features&tags=None, cm
         env = env,
     )
 
+def export_large_file(name:str, src:str, visibility:list=None, binary:bool=False, test_only:bool&testonly=False):
```

Comment:
I think it would be nice to support system sources too, what do you think?

Reply:
I don't know. I feel like a certain level of awkwardness around system sources is good. I'm on the fence about this rule to be honest. I almost feel like this should be difficult to avoid people doing it without due consideration.